config/initializers/active_job_hooks.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+require "active_support/core_ext/hash"
```

Comment:
Since this one appears to be loaded from standard libs, wouldn't file be loaded by itself or we need to do here. I guess we do but have to ask :)